app/services/discovery_engine/sync/delete.rb

```diff
@@ -2,7 +2,7 @@ module DiscoveryEngine::Sync
   class Delete < Operation
     include Locking
 
-    def initialize(content_id = nil, payload_version: nil, client: nil)
+    def initialize(content_id, payload_version: nil, client: nil)
```

Comment:
Ah, this makes more sense. Why was `content_id` made optional?

Reply:
Ah yes, it was briefly optional as an intermediate step of the refactoring when I allowed both `#initialize` and `#call` to receive these arguments!